acceptance/suites/upgrade_tests/10_upgrade.rb

```diff
@@ -23,6 +26,12 @@
   on(master, puppet("resource service puppetserver ensure=running"))
 end
 
+step "Upgrade Puppet agents" do
+  nonmaster_agents.each do |agent|
```

Comment:
It's probably redundant since the puppetserver upgrade should transitively upgrade puppet-agent as well, since the newer puppetserver will have a newer puppet-agent dependency.  Might be nice to not upgrade the agent on the master so that we can prove the agent does get upgraded as expected just by upgrading puppetserver.